packages/runtime/runtime-definitions/api-report/runtime-definitions.legacy.alpha.api.md

```diff
@@ -93,7 +93,7 @@ export interface IContainerRuntimeBase extends IEventProvider<IContainerRuntimeB
 // @alpha @sealed (undocumented)
 export interface IContainerRuntimeBaseEvents extends IEvent {
     (event: "batchBegin", listener: (op: Omit<ISequencedDocumentMessage, "contents">) => void): any;
-    (event: "batchEnd", listener: (error: any, op: Omit<ISequencedDocumentMessage, "contents">) => void): any;
+    (event: "batchEnd", listener: (error: unknown, op: Omit<ISequencedDocumentMessage, "contents">) => void): any;
```

Comment:
API change is on a `@sealed` interface that Fluid consumers have no need to implement.